docs/device_api.md

```diff
@@ -525,7 +525,7 @@ In the case of complex commands requiring parameters, the `commandValue` could b
 
 This example will tell the device to reply to a ping message with the provided params. 
 
-Note that you cannot use [forbidden character for Context Broker](https://fiware-orion.readthedocs.io/en/master/user/forbidden_characters/index.html) as part of command parameters, because that would result in a 400 Bad Request error and at CB and the triggering of the command would be never progressed.
+Note that you cannot use [forbidden character for Context Broker]( https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#general-syntax-restrictions) as part of command parameters, because that would result in a 400 Bad Request error and at CB and the triggering of the command would be never progressed.
```

Comment:
```suggestion
Note that you cannot use [forbidden character for Context Broker](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#general-syntax-restrictions) as part of command parameters, because that would result in a 400 Bad Request error and at CB and the triggering of the command would be never progressed.
```